web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
Sylvain,
Whatever the rules define in guidelines, what is the gain to put 9 lines in another file whereas the whole file is only 61 lines ?
I have a better developer experience (#productivity) when I have a reasonable number of lines in my files than switch between tabs (search, replace, etc...).
Too many rules kill the rules. Python != Java. Don't you think ?